src/js/components/Home/Countries.js

```diff
@@ -0,0 +1,60 @@
+import React, {Component} from "react";
+import Map from "./Map";
+import ReactFlagsSelect from 'react-flags-select';
+import  countries from 'react-flags-select/es/countries';
+import Geocode from "react-geocode";
+
+Geocode.setApiKey("AIzaSyBiCz9njYP8nYxEUEQLzltCM8kAnm7z6To");
+
+class Mapbuttons extends Component {
+    constructor() {
+        super()
+        this.state = {
+            center: [0, 0]
+        }
+        this.getCountryCode = this.getCountryCode.bind(this);
+        this.setCountry = this.setCountry.bind(this);
+    }
+
+    setCountry(countryCode) {
+        this.getCountryCode( countries[countryCode]);
+    }
+
+    getCountryCode(country) {
```

Comment:
You can wrap your method inside the componentDidMount function to ensure that the component is done  mounting to the ReactDOM